PriceCalculatorKata.Common/FloatExtensions.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System.Globalization;
+
+namespace PriceCalculatorKata.Common;
+
+public static class FloatExtensions
+{
+    public static bool IsValid(this float source)
```

Comment:
Is the name IsValid here clear? remember that this is an extension method, where it will be called directly on an object without any context given.

Reply:
> Is the name IsValid here clear? remember that this is an extension method, where it will be called directly on an object without any context given.

I guess I was just looking at input values being "valid" based on the given requirements. I've fixed the names to describe what each method does better.